apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -844,12 +886,29 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
       assert max_amount_spendable_in_single_tx(alice.addr, @eth) >= payment
     end
 
-    @tag fixtures: [:alice, :bob, :more_utxos]
-    test "advice on merge does not merge single utxo", %{alice: alice, bob: bob} do
+    @tag fixtures: [:alice, :bob, :more_utxos, :test_server]
+    test "advice on merge does not merge single utxo", %{alice: alice, bob: bob, test_server: context} do
       max_spendable = max_amount_spendable_in_single_tx(alice.addr, @eth)
 
       payment = max_spendable + 1
 
+      fee_response = %{
+        @str_tx_type => [
+          %{
+            "currency" => @default_fee_currency,
+            "amount" => 0,
```

Comment:
not sure I understand why amount is 0 here?

Reply:
EDIT: ok it used to be 0 before. Like in other similar comments - try not customizing this to 0, it seems like it should work with `@fee_response`.